client/rest-high-level/src/test/java/org/opensearch/client/SearchIT.java

```diff
@@ -793,7 +796,15 @@ public void testSearchWithPit() throws Exception {
                 assertThat(((Number) hit.getSortValues()[0]).longValue(), equalTo(counter++));
             }
         } finally {
-            // TODO : Delete PIT
+            List<String> pitIds = new ArrayList<>();
+            pitIds.add(pitResponse.getId());
+            DeletePitRequest deletePitRequest = new DeletePitRequest(pitIds);
+            DeletePitResponse deletePitResponse = execute(
+                deletePitRequest,
+                highLevelClient()::deletePit,
+                highLevelClient()::deletePitAsync
+            );
+            assertTrue(deletePitResponse.isSucceeded());
```

Comment:
let's verify that searchservice map of `activeReader` is empty on each node

Reply:
we can use 'list all pits api' to verify active pits as part of 'list all pits' pr.